api-spec/adminapi_peers_mesh.yaml

```
@@ -0,0 +1,20 @@
+get:
+  tags:
+    - admin
+  summary: Get peers by service
+  description: Returns information about relay peers in node's direct mesh, grouped by shards.
+  operationId: getMeshPeers
+  responses:
+    '200':
+      description: Successful service peers response
```

```suggestion
      description: Successful relay peers response
```

maybe? Same for other endpoints